concepticondata/conceptlists/Clark-2008-757.tsv

```diff
@@ -0,0 +1,789 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Clark-2008-757-1	1	bear	1820	BEAR
+Clark-2008-757-2	2	frog	503	FROG
+Clark-2008-757-3	3	monkey	1350	MONKEY
+Clark-2008-757-4	4	mosquito	1509	MOSQUITO
+Clark-2008-757-5	5	pig	1337	PIG
+Clark-2008-757-6	6	snail	1543	SNAIL
+Clark-2008-757-7	7	snake	730	SNAKE
+Clark-2008-757-8	8	spider	843	SPIDER
+Clark-2008-757-9	9	tiger	846	TIGER
+Clark-2008-757-10	10	turtle	112	TURTLE
+Clark-2008-757-11	11	earthworm	2350	EARTHWORM
+Clark-2008-757-12	12	bee	665	BEE
+Clark-2008-757-13	13	bird	937	BIRD
+Clark-2008-757-14	14	buffalo	934	BUFFALO
+Clark-2008-757-15	15	cat	1208	CAT
+Clark-2008-757-16	16	chicken	1318	CHICKEN
+Clark-2008-757-17	17	dog	2009	DOG
+Clark-2008-757-18	18	fish	227	FISH
+Clark-2008-757-19	19	fly	1441	FLY (MOVE THROUGH AIR)
+Clark-2008-757-20	20	animal	619	ANIMAL
+Clark-2008-757-21	21	deer	1936	DEER
+Clark-2008-757-22	22	rabbit	1136	RABBIT
+Clark-2008-757-23	23	porcupine	2510	PORCUPINE
+Clark-2008-757-24	24	cow	1007	COW
+Clark-2008-757-25	25	elephant	1290	ELEPHANT
+Clark-2008-757-26	26	duck	1360	DUCK
+Clark-2008-757-27	27	house lizard	632	LIZARD
+Clark-2008-757-28	28	crocodile	1857	CROCODILE
+Clark-2008-757-29	29	tadpole		
+Clark-2008-757-30	30	louse (head)	310	HEAD LOUSE
+Clark-2008-757-31	31	termite	883	TERMITE
+Clark-2008-757-32	32	cockroach	1575	COCKROACH
+Clark-2008-757-33	33	butterfly	1791	BUTTERFLY
+Clark-2008-757-34	34	scorpion	1538	SCORPION
+Clark-2008-757-35	35	ant	587	ANT
+Clark-2008-757-36	36	crow	913	CROW
+Clark-2008-757-37	37	barking deer	3152	MUNTJACS
+Clark-2008-757-38	38	sambar deer	2456	LARGE WILD HERBIVORE
+Clark-2008-757-39	39	dragon	2038	DRAGON
+Clark-2008-757-40	40	dragon fly	2352	DRAGONFLY
+Clark-2008-757-41	41	duck1	1360	DUCK
+Clark-2008-757-42	42	duck2		
+Clark-2008-757-43	43	eagle	1905	EAGLE
+Clark-2008-757-44	44	flea	232	FLEA
+Clark-2008-757-45	45	goat	2684	CAPRINE (GOAT OR SHEEP)
+Clark-2008-757-46	46	bat	1793	BAT
+Clark-2008-757-47	47	grasshopper	607	GRASSHOPPER
+Clark-2008-757-48	48	horse	615	HORSE
+Clark-2008-757-49	49	insect	620	INSECT
+Clark-2008-757-50	50	leech, dry	2273	LEECH
+Clark-2008-757-51	51	leech, wet	949	WATER LEECH
+Clark-2008-757-52	52	lizard	632	LIZARD
+Clark-2008-757-53	53	louse, head	310	HEAD LOUSE
+Clark-2008-757-54	54	bore	1501	BORE
+Clark-2008-757-55	55	piglet	2564	PIGLET
+Clark-2008-757-56	56	pig sow	1351	SOW (FEMALE PIG)
+Clark-2008-757-57	57	caterpillar	2247	CATERPILLAR
+Clark-2008-757-58	58	rat	2139	MUROID (MOUSE OR RAT)
```

Comment:
I will keep this link with MUROID (MOUSE OR RAT), because Chen's data also links to the same concept. 
Clark's data has "rat" (du²¹⁴) and Chen's data has 老鼠 (Kim Mun : du³⁵)